riva/clients/asr/riva_asr_client.cc

```diff
@@ -457,13 +459,17 @@ main(int argc, char** argv)
   std::shared_ptr<grpc::Channel> grpc_channel;
   try {
     std::shared_ptr<grpc::ChannelCredentials> creds;
-    if (FLAGS_ssl_cert.size() > 0) {
-      auto cacert = riva::utils::files::ReadFileContentAsString(FLAGS_ssl_cert);
-      grpc::SslCredentialsOptions ssl_opts;
-      ssl_opts.pem_root_certs = cacert;
-      LOG(INFO) << "Using SSL Credentials";
-      creds = grpc::SslCredentials(ssl_opts);
-    } else {
+    if (FLAGS_ssl_cert.size() > 0 || FLAGS_use_ssl) {
+        LOG(INFO) << "Using Secure Server Credentials";
+        grpc::SslCredentialsOptions ssl_opts;
+        if (FLAGS_ssl_cert.size() > 0) {
+            LOG(INFO) << "Using SSL Credentials";
+            auto cacert = riva::utils::files::ReadFileContentAsString(FLAGS_ssl_cert);
+            ssl_opts.pem_root_certs = cacert;
+        }
+        creds = grpc::SslCredentials(ssl_opts);
+    }
+    else {
```

Comment:
did you run pre-commit hook to fix formatting?
otherwise LGTM.

Reply:
I tried to run formatting, but ran into some errors.
See #51 for details.